packages/google_maps_flutter/google_maps_flutter/ios/Classes/GoogleMapController.m

```diff
@@ -172,20 +172,24 @@ - (void)onMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
   } else if ([call.method isEqualToString:@"map#waitForMap"]) {
     result(nil);
   } else if ([call.method isEqualToString:@"map#takeSnapshot"]) {
-    if (_mapView != nil) {
-      UIGraphicsImageRendererFormat* format = [UIGraphicsImageRendererFormat defaultFormat];
-      format.scale = [[UIScreen mainScreen] scale];
-      UIGraphicsImageRenderer* renderer =
-          [[UIGraphicsImageRenderer alloc] initWithSize:_mapView.frame.size format:format];
-
-      UIImage* image = [renderer imageWithActions:^(UIGraphicsImageRendererContext* context) {
-        [_mapView.layer renderInContext:context.CGContext];
-      }];
-      result([FlutterStandardTypedData typedDataWithBytes:UIImagePNGRepresentation(image)]);
+    if (@available(iOS 10.0, *)) {
```

Comment:
Matched the pattern at https://github.com/flutter/plugins/blob/5ca81a351d34558380e2f9f9ec9f6b6205435a49/packages/webview_flutter/ios/Classes/FLTCookieManager.m#L45.

Reply:
Thanks for catching this!